src/pages/trainer/training/edit/actions/updateEditorCursor.ts

```diff
@@ -0,0 +1,10 @@
+import {trainerActionEnums} from '../../../../../common/actionEnums/trainer';
+import {textAreaTool} from '../../../../../common/ui/tools/textAreaTool';
+
+export const updateEditorCursorAction = (editor: HTMLTextAreaElement, cursorStartPosition: number) => {
+  textAreaTool.placeCursor(editor, cursorStartPosition);
+
+  return {
+    type: trainerActionEnums.UPDATE_EDITOR_CURSOR,
+  };
```

Comment:
Same here, I think this is pure UI, and eve this action is not parsed by the reducer?